assets/js/sketch.js

```diff
@@ -29,19 +29,36 @@ function setup() {
   rectMode(CORNER);
 }
 
+defInk = "#264180";//Default ink
+
+function chooseBlue(){
+  defInk = "#264180"
+}
+
+function chooseRed(){
```

Comment:
Add comments or some function description please, later on it would become a big chunk of code and difficult to debug

Reply:
ok I'll be